src/Main.java

```diff
@@ -1,5 +1,70 @@
 public class Main {
     public static void main(String[] args) {
-        System.out.println("Hello world!");
+        System.out.println("Задача 1");
+        int contribution = 15000;
+        int total = 0;
+        int mounth = 0;
+        while (total <= 2_459_000){
+           mounth = mounth + 1;
+           total = total + contribution;
+           total = total + total/100;
+           System.out.println("Месяц " + mounth + " Сумма накоплений равна " + total + " Рублей");
+       }
+        System.out.println("Для того, чтобы накопить " + total + " С ежемесячным вкладом " + contribution + " Необходимо " + mounth + " Месяцев");
+        System.out.println("Задача 2");
+        int numbersF = 0;
+        while (numbersF < 10){
+            numbersF = numbersF + 1;
+            System.out.print(numbersF + " ");
+        }
+        System.out.println();
+        for (int numbersB = 10; numbersB > 0; numbersB--){
+            System.out.print(numbersB + " ");
+        }
+        System.out.println();
+        System.out.println("Задание 3");
+        int Y = 12_000_000;
```

Comment:
```suggestion
        int y = 12_000_000;
```
название переменных всегда с маленькой буквы пишем